saleor/graphql/product/mutations/products.py

```diff
@@ -846,3 +851,161 @@ def mutate(cls, root, info, image_id, variant_id):
                     variant_image.delete()
         return VariantImageUnassign(
             product_variant=variant, image=image, errors=errors)
+
+class ProductVariantDigitalInput(graphene.InputObjectType):
```

Comment:
I think we don't have to reveal in names of this type that it is related to variants underneath. The API would look cleaner if we had simply `DigitalContentInput`.

Reply:
Also, in GraphQL types, we have the `DigitalContent` type and mutations should also use that naming pattern for better consistency.

Reply:
What do you think about moving all digital content related mutations to a separate file?